tools/src/test/java/org/apache/rocketmq/tools/command/connection/ConsumerConnectionSubCommandTest.java

```diff
@@ -99,4 +69,30 @@ public void testExecute() throws SubCommandException {
             ServerUtil.parseCmdLine("mqadmin " + cmd.commandName(), subargs, cmd.buildCommandlineOptions(options), new PosixParser());
         cmd.execute(commandLine, options, null);
     }
+
+    private ServerResponseMocker startNameServer() {
```

Comment:
there's so many startNameServer() method that exists in multiple classes， So how about putting the mock to the nameserver in the same test tool class to avoid so much duplication code?

Reply:
When coding, I also realized that the code repetition of startNameServer() is relatively high. I wanted to retain the flexibility of this method.

However, in most scenarios, startNameServer() can indeed use the default configuration. I will extract a NameServer of the default type, If other test cases have specific requirements, rewrite their own logic

Reply:
@duhenglucky   done